replayer/connemara_replay.pl

```diff
@@ -259,65 +259,6 @@ sub get_prep_statement_object
     return $sth;
 }
 
-# This is called when idling. We create missing indexes, in case we got a FK creation for instance
-sub create_missing_indexes
```

Comment:
Can you please give a reason of why you are splitting out reindexing job into a shell script?